docs/FAQ.md

```diff
@@ -4,6 +4,23 @@ This is a collection of questions that often come up as people are getting start
 If you tried to go through the [Getting Started guide](https://hexdocs.pm/nerves/getting-started.html) or some of the [example projects](https://github.com/nerves-project/nerves_examples) and got stuck, hopefully one of the following answers will help.
 If not, please let us know in the #nerves channel on [the Elixir-Lang Slack](https://elixir-slackin.herokuapp.com/), or [create an Issue or Pull Request to improve this documentation](https://github.com/nerves-project/nerves/tree/main/docs).
 
+## Where can persistent data be stored?
+
+By default the `/data` partition both read/writeable and is not overwritten when new firmware is pushed to the device.
+
+## How can I apply a firmware update manually?
+
+```
+iex> cmd("fwup -i /data/known_good.fw --apply --task complete " <>
+  "--no-unmount -d #{Nerves.Runtime.KV.get("nerves_fw_devpath")}")
```

Comment:
I built this command by looking at what https://github.com/nerves-project/nerves_firmware_ssh/blob/86e43429c87544c7d18a478bfd200df5667466ff/lib/nerves_firmware_ssh/fwup.ex does. However, this command does not work as-is (testing so far on nerves_system_vultr). On boot I get the following errors:
```
19:49:53.921 [error] FAT-fs (vda1): codepage cp437 not found
19:49:53.921 [warn]  erlinit: Cannot mount /dev/vda1 at /mnt: Invalid argument
19:49:53.921 [error] EXT4-fs (vda4): ext4_check_descriptors: Block bitmap for group 0 not in group (block 4294967295)!
19:49:53.921 [error] EXT4-fs (vda4): group descriptors corrupted!
19:49:53.921 [warn]  erlinit: Cannot mount /dev/vda4 at /root: No error information
19:49:53.924 [info]  mount: mounting /dev/vda4 on /root failed: No error information
19:49:53.925 [warn]  heart: kernel watchdog activated (interval 5s)
19:49:53.925 [error] EXT4-fs (vda4): ext4_check_descriptors: Block bitmap for group 0 not in group (block 4294967295)!
19:49:53.925 [error] EXT4-fs (vda4): group descriptors corrupted!
19:49:53.925 [warn]  Ignoring non-zero exit status from mount ["-t", "ext4", "-o", "rw", "/dev/vda4", "/root"]
```

Also, I believe we'll need to tell the user to run `reboot/0`.

Reply:
If you change the task to "upgrade", it should work.

Reply:
Ah, that does work :+1:. Didn't expect "upgrade" to be a task based on the `fwup -l` output:
```
fwup -l -i good.fw
complete
upgrade.a
upgrade.b
upgrade.unexpected
upgrade.wrongplatform
```
I thought you would need to do `upgrade.a` or b

Reply:
Yeah, the task passed on the commandline is a prefix match for the first applicable task in the fwup.conf file. That lets you force fwup to upgrade the A or B partition or let fwup decide which is the right one.

Reply:
Okay, that makes sense :+1: